### ISSUE TITLE
Content editing is difficult

### ISSUE THREAD

**Issue by jamiefolsom** [hofstra/coyote#8]
- For routine tasks, use [Site Leaf](https://manage.siteleaf.com/sites/5902a1d09be5f52fbf89c9e3/pages). Must be upgraded to a non-free account, to allow for volume of content. Check with Jamie/Nick about that upgrade.
- For tags, which have to be edited both within source posts, and within tag posts: can we migrate tags to data files, or collections, so that they are more "relational"? See: http://jekyllrb.com/docs/collections/ and http://jekyllrb.com/docs/datafiles/

### Pain points
- Metadata (frontmatter) editing (punctuation has a different meaning
- Content editing is not too bad
- Adding a new source is not too bad
- Easily add some content below the image or the artifact (above or below)
- Article: add a brief summary

Basically -- ease of editing is crucial

**Comment by jamiefolsom** [comment]
### What I've found out

- [ ] Tags are inserted into source posts as strings

### What I think we should do about it

- [ ] I think we should use Jekyll collections instead

@tezcatlipoca what do you think?

**Comment by feralresearch** [comment]
@jamiefolsom 

Collections would be fine I think.

Alternatively, on my own sites, I make use of the datafiles directory and a short script for deployment which prefaces the 'bundle exec...' with curl statements to retrieve the data off of the internet. This means I can just edit a spreadsheet or text file (could probably be hosted on google docs I think) and then it gets rolled into Jekyll at "compile time." It's low-tech but it works nicely. The script could also create the requisite .md files for our site if needed. 

I have no idea if something like this would work here - a roundabout way of asking who is doing the editing and deployment and how. 

I'll attach an example.

[deploy.sh.zip](https://github.com/hofstra/coyote/files/1388850/deploy.sh.zip)
